exim.yaml

```diff
@@ -27,7 +27,6 @@ environment:
       - build-base
       - ca-certificates-bundle
   accounts:
-    runs: exim
```

Comment:
run-as?

Reply:
This setting is ignored by Melange.  No point in having it.